{"error": "CVE_RECORD_DNE", "message": "The cve record for the cve id does not exist."}